cpp/include/cudf/detail/null_mask.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2021, NVIDIA CORPORATION.
+ * Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
```suggestion
 * Copyright (c) 2019-2022, NVIDIA CORPORATION.
```
The copyright start year should remain unchanged.